harness/determined/exec/prep_container.py

```diff
@@ -263,6 +302,11 @@ def do_proxy(sess: api.Session, allocation_id: str) -> None:
         ),
     )
 
+    # Notify the Determined Master that the container is running.
```

Comment:
i think this line of the comment is redundant

Reply:
OK, I can remove next time I change this file.